chat_shell/chat_shell/compression/token_counter.py

```
@@ -236,3 +237,59 @@ def is_over_limit(self, messages: list[dict[str, Any]], context_limit: int) -> b
             True if messages exceed the limit
         """
         return self.count_messages(messages) > context_limit
+
+    def get_context_window(self) -> int:
+        """Get the context window size for the current model.
+
+        Returns:
+            Context window size in tokens
+        """
+        # Context window sizes for common models
+        context_windows = {
```

Ruff (0.14.10)

Local variable `context_windows` is assigned to but never used

Remove assignment to unused variable `context_windows`

(F841)